gnucash/import-export/aqb/gnc-ab-utils.c

```diff
@@ -1062,13 +1062,13 @@ bal_accountinfo_cb(AB_IMEXPORTER_ACCOUNTINFO *element, gpointer user_data)
 #else
             time64 secs = GWEN_Time_toTime_t(ti);
 #endif
-            booked_tt = gnc_time64_get_day_neutral(secs);
+            booked_tt = gnc_time64_get_day_end(secs);
```

Comment:
This is probably not what caused the problem with [Bug 797673](https://bugs.gnucash.org/show_bug.cgi?id=797673) because as I noted in [comment 17](https://bugs.gnucash.org/show_bug.cgi?id=797673#c17) the reconciliation date on the two errant splits was before neutral time.

One could argue that reconciling to a downloaded balance should reflect the time of the download: The balance might change later in the day and the user might do another reconcile. The alternative I suppose would be to use end-of-day, resulting in a new reconciled balance for what becomes as far as GnuCash is concerned the same reconcile.

Reply:
I'll forgo this bugfix because I don't understand it enough.